File: src/main/java/com/gangofconnectfour/powerfourservice/api/in/GameDtoIn.java
package com.gangofconnectfour.powerfourservice.api.in;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GameDtoIn {

    private Long id;
    private int aiDifficulty = 0;
    private Boolean versusAi = Boolean.FALSE;

}

File: src/main/kotlin/com/gangofconnectfour/powerfourservice/api/in/GameDtoIn.kt
package com.gangofconnectfour.powerfourservice.api.`in`

class GameDtoIn {

    var token : String = ""
    var uuid : String = ""
    var aiDifficulty : Int = 0
    var versusAi : Boolean = false

    constructor()
    constructor(token: String, uuid: String, aiDifficulty: Int, versusAi: Boolean) {
        this.token = token
        this.uuid = uuid
        this.aiDifficulty = aiDifficulty
        this.versusAi = versusAi
    }


}